cli/src/commands/lock_files.rs

```diff
@@ -137,6 +147,7 @@ mod tests {
             ("tests/fixtures/requirements.txt", PackageType::PyPi),
             ("tests/fixtures/Pipfile", PackageType::PyPi),
             ("tests/fixtures/Pipfile.lock", PackageType::PyPi),
+            ("tests/fixtures/Poetry.lock", PackageType::PyPi),
```

Comment:
```suggestion
            ("tests/fixtures/poetry.lock", PackageType::PyPi),
```
...to match the name in source and also the way it gets generated by `poetry` itself.